Apps/Sandcastle/gallery/Video.html

```diff
@@ -75,7 +75,8 @@
 
     synchronizer = new Cesium.VideoSynchronizer({
         clock : viewer.clock,
-        element : videoElement
+        element : videoElement,
+        tolerance : 0.1
```

Comment:
Let's remove this from the example, we should provide good defaults instead of immediately overriding things in the demo.

As for the defaults, I think for now it's OK to have a single tolerance value, and for the video to be choppy in reverse (reverse overall is an edge case anyway).  The problem with having a really low tolerance value is that seeking constantly starts to eat up CPU.  If we ever have a good use case for it, we can add a reverseTolerance in the future.